bigtable/client/table.h

```diff
@@ -167,6 +173,18 @@ class Table {
    */
   void BulkApply(BulkMutation&& mut);
 
+  /**
+   * Reads data from a set of rows.
+   *
+   * @param row_set the rows to read from.
+   *
```

Comment:
No blanks between the `@param` sections?

Reply:
done